src/ansiblelint/loaders.py

```diff
@@ -24,4 +29,29 @@ def yaml_from_file(filepath: str | Path) -> Any:
         return yaml_load(content)
 
 
-__all__ = ["yaml_from_file", "yaml_load", "yaml_load_safe", "YAMLError"]
+def load_ignore_txt(filepath: str | Path = IGNORE_TXT) -> dict[str, set[str]]:
+    """Return a list of rules to ignore."""
+    result = defaultdict(set)
+    if os.path.isfile(filepath):
+        with open(str(filepath), encoding="utf-8") as content:
+            _logger.debug("Loading ignores from %s", filepath)
+            for line in content:
+                entry = line.split("#")[0].rstrip()
+                if entry:
+                    try:
+                        path, rule = entry.split()
```

Comment:
if path.endwith(':'), path = path[0:-1] ?

Reply:
Nope, only spaces supported for now. No plan to add unneeded complexity.